dask_kubernetes/core.py

```diff
@@ -14,20 +17,207 @@
     yaml = False
 
 import dask
-from distributed.deploy import LocalCluster, Cluster
+from distributed.deploy import SpecCluster, ProcessInterface
 from distributed.comm.utils import offload
-import kubernetes
+from distributed.utils import Log, Logs
+from distributed.scheduler import Scheduler as LocalScheduler
```

Comment:
```suggestion
from distributed import Scheduler as LocalScheduler
```

Reply:
We might also `import dask.distrtibuted` and then use the fully namespaced `dask.distributed.Scheduler` term where we would use `LocalScheduler`